tests/main.tf

```diff
@@ -228,10 +238,10 @@ module "elasticsearch" {
   credstash_kms_key_arn       = ""
   credstash_reader_policy_arn = ""
   elasticsearch_dns_name      = ""
-  elasticsearch_version       = ""
-  internal_alb                = ""
+  elasticsearch_version       = "0"
+  internal_alb                = {}
```

Comment:
Is this variable now a map?

Reply:
It is:
```
variable "internal_alb" {
  type = map(string)

  description = <<DOC
Information on already existing Application LB or settings for a Classic LB to
be deployed:
  * security_group_id (required)
  * deploy_elb (optional, default: false) - if set to true ELB will be deployed
    and used instead of an ALB.
  * deploy_elb_internal (optional, default: true) - if set to false ELB will be
    deployed as an external one.
  * deploy_elb_cross_zone (optional, default: true) - if set to false cross AZ
    for ELB will be off.
Below are only required if `deploy_elb` is set to `false` or omitted:
  * arn - ALB ARN.
  * dns_name - host-header for ALB listener rule.
  * zone_id - not used directly, but will be returned in the `lb` output map.
DOC

}
```

But it makes more sense to have an `object` here with one visible problem: HCL doesn't allow optional object attributes.